hastexo/hastexo.py

```diff
@@ -232,10 +240,32 @@ def parse_xml(cls, node, runtime, keys, id_generator):
 
         return block
 
-    def author_view(self, context=None):
-        """ Studio View """
-        msg = u"This XBlock only renders content when viewed via the LMS."
-        return Fragment(u'<em>%s</em></p>' % msg)
+    @property
+    def allowed_nested_blocks(self):
+        """
+        Returns a list of allowed nested blocks.
+
+        """
+        additional_blocks = []
+        try:
+            from xmodule.video_module.video_module import VideoDescriptor
+            _spec = NestedXBlockSpec(
+                VideoDescriptor, category="video", label=u"Video"
+            )
+            additional_blocks.append(_spec)
+        except ImportError:
+            pass
```

Comment:
In this case I'm curious _how_ an import from `xmodule.video_module.video_module` would fail unless edx-platform is really really badly misconfigured. Maybe log this at the warning level? As in:
```python
except ImportError:
  logging.warning("Unable to import VideoDescriptor", exc_info=True)
```

Reply:
This is in case the XBlock is running from the workbench.  I realize we don't really support this anyway, but it is used as an example in `olx-utils`, and I figured it was harmless to keep the suggested behavior.